integration-tests/contract-batch.spec.ts

```diff
@@ -2,6 +2,7 @@ import { CONFIGS } from './config';
 import { ligoSample, ligoSampleMichelson } from './data/ligo-simple-contract';
 import { managerCode } from './data/manager_code';
 import { MANAGER_LAMBDA, OpKind } from '@taquito/taquito';
+import { BatchOperation } from 'taquito/dist/types/operations';
```

Comment:
the import path doesn't seem right.

Reply:
good catch, looks like auto import didn't get the correct path